common/models/templates/src/trie-model.ts

```diff
@@ -161,6 +161,116 @@
     public wordbreak(context: Context): USVString {
       return this.getLastWord(context.left);
     }
+
+    public getRootTraversal(): LexiconTraversal {
+      return new TrieModel.Traversal(this._trie['root'], '');
+    }
+
+    private static Traversal = class implements LexiconTraversal {
+      prefix: String;
+      root: Node;
```

Comment:
Isn't this always an arbitrary node? It doesn't necessarily have to be the root node of the trie?

```suggestion
      node: Node;
```

Reply:
Yes, it is an arbitrary node... the root of its sub-trie.  Parent nodes of the instance stored in `root` can't be revisited from a `LexiconTraversal` instance.

Perhaps documenting that more explicitly is in order.